config/environment.js

```diff
@@ -25,8 +25,8 @@ module.exports = function(environment) {
             // when it is created
         },
         SHARE: {
-            baseUrl: process.env.SHARE_BASE_URL || 'https://staging-share.osf.io/',
-            searchUrl: process.env.SHARE_SEARCH_URL || 'https://staging-share.osf.io/api/search/abstractcreativework/_search'
+            baseUrl: process.env.SHARE_BASE_URL || 'https://share.osf.io/',
```

Comment:
I assume we want to do this, but can you confirm that this is an intentional change (vs accidental commit)?

Not sure how staging gets its config, so we should verify that the rule change doesn't affect QA (eg if they think that search is pointing at one server, and it suddenly is looking at somewhere else instead).